db/migrations/20190807230019-add-indexes.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addIndex('Subscriptions', {
+      fields: ['gitHubInstallationId', 'jiraHost'],
+      name: 'index-on-installation-and-jira-host'
```

Comment:
I'm new to Postgres so take this with a grain of salt. After some Googling, I found some [naming conventions](https://stackoverflow.com/questions/4107915/postgresql-default-constraint-names/4108266#4108266) that seem to match internal Postgres indexes (`SELECT * FROM pg_indexes`).

What do you think of this name?

```suggestion
      name: 'Subscriptions_gitHubInstallationId_jiraHost_idx'
```

Reply:
Works for me! I was doing a mental conversion between how we define them in `github/github` and what that might look like in JS, but making them look more like Postgres is probably better.